readme.md

```diff
@@ -1,7 +1,7 @@
 # Stars block
 Replace `<your_account>` with your Github username and copy the links to Pull Request description:
-- [DEMO LINK](https://<your_account>.github.io/layout_stars/)
-- [TEST REPORT LINK](https://<your_account>.github.io/layout_stars/report/html_report/)
+- [DEMO LINK](https://Deobp.github.io/layout_stars/)
+- [TEST REPORT LINK](https://Deobp.github.io/layout_stars/report/html_report/)
```

Comment:
Add this links to PR description 
![image](https://github.com/mate-academy/layout_stars/assets/70009205/72224622-5396-4974-8207-46237a9da268)
![image](https://github.com/mate-academy/layout_stars/assets/70009205/38062dfb-f6f0-4665-8cb4-6379af854dc0)